python/tvm/runtime/profiling/__init__.py

```diff
@@ -47,6 +47,36 @@ def csv(self):
         """
         return _ffi_api.AsCSV(self)
 
+    def json(self):
+        """Convert this profiling report into JSON format.
+
+        Example output:
+
+        .. code-block:
+
+           {"calls":
```

Comment:
See comment on C++ above, would be great if this were pretty-printed for readability (in the comment itself, no need for the actual JSON output).